back/engines/commercial/custom_maps/app/services/custom_maps/side_fx_map_config_service.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module CustomMaps
+  class SideFxMapConfigService < BaseSideFxService
+    include SideFxHelper
+
+    def before_create(_map_config, _user); end
+
+    def before_update(_map_config, _user); end
+
+    def before_destroy(_map_config, _user); end
```

Comment:
Do we need these to be defined at all if we are inheriting from BaseSideFxService? Should work without them

Reply:
We do this quite often, and I remember it being discussed in CDM (but don't remember any clear consensus). I'm happy to remove them.